openapi3.yaml

```diff
@@ -654,6 +654,31 @@ components:
       example: 'Pending'
     job:
       type: object
+      required:
```

Comment:
if you force all output only fields as required rename the model from job to jobResponse

Reply:
done